__init__.py

```diff
@@ -194,6 +197,7 @@ def handle_forecast(self, message):
             report = self.__initialize_report(message)
 
             # Get a date from spoken request
+            LOG.debug("lang %s" % self.lang)
```

Comment:
Not a critical thing but I think it's better to use the new style `'lang {}'.format(self.lang)` instead of the old % notation.